cluster/saltbase/salt/kubelet/init.sls

```diff
@@ -31,19 +31,6 @@
     - mode: 400
     - makedirs: true
 
-#
```

Comment:
I would also like to rip it out of https://github.com/GoogleCloudPlatform/kubernetes/blob/master/cmd/kubelet/app/server.go FWIW.